### ISSUE TITLE
Deprecate on Hackage

### ISSUE THREAD

**Issue by andreasabel** [noughtmare/haskell-lsp-client#8]
Looks like this package has died, I think it should be deprecated on hackage.  Is there a replacement project?

**Comment by noughtmare** [comment]
I guess it depends on your definition of died. I don't have any plans for this package personally, but I'd gladly help any new contributors.

As for alternatives, I see there is `lsp-client`, but I don't know anything about it.

**Comment by andreasabel** [comment]
Yes, I wondered whether `lsp-client` was the successor, but does not seem to be so.  Thanks for the answer!

What Stackage concerns, `haskell-lsp-client` is still listed, but disabled.  
I was wondering what to do about it:
- https://github.com/commercialhaskell/stackage/pull/7185/files

I have left it there in disabled state, but if you have no plans to update it, maybe it can go completely?

**Comment by noughtmare** [comment]
I'm fine with it being removed completely from Stackage. Apologies for the inconvenience, this is the first and only Stackage package I've added, so I'm not quite sure what my duties are/were.

I've now deprecated the package on Hackage.

**Comment by andreasabel** [comment]
No apologies please.  I just came by here when looking after Stackage maintenance for `lsp`.  Thanks for taking care!
